foundations/installations/installations.md

```diff
@@ -352,7 +352,11 @@ On Windows there are three primary ways to open WSL2.
 - If you search for Ubuntu in the application search bar you should see a application titled Ubuntu; open it to start a new terminal session.
 
 <div class="lesson-note lesson-note--tip" markdown="1">
-You might notice when you open WSL2 via Windows Terminal, you'll see a window with a different color scheme and a different icon compared to opening a terminal through Ubuntu in your applications. This is because Windows Terminal comes with a default color scheme for Ubuntu meant to emulate how a real Ubuntu terminal looks. This difference is purely cosmetic, and there is no practical difference between the two.
```

Comment:
I'm not sure why this has been removed?

Reply:
This needs to be addressed. Why are you replacing this entire paragraph with your own? I don't understand.